subsys/net/lib/wifi_credentials/Kconfig

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig WIFI_CREDENTIALS
+	bool "WIFI credentials management"
+	help
+	  Enable WiFi credentials management subsystem.
+
+if WIFI_CREDENTIALS
+
+module = WIFI_CREDENTIALS
+module-str = wifi_credentials
+source "subsys/logging/Kconfig.template.log_config"
+
+choice WIFI_CREDENTIALS_BACKEND
+	prompt "wifi credentials backend"
+	default WIFI_CREDENTIALS_BACKEND_SETTINGS
+	help
+	  This library can either use the PSA Protected Storage backend or the settings backend.
```

Comment:
Nit: `Zephyr settings subsysem` makes it a bit more clear exactly what settings means

Reply:
done